_includes/cli.md

```diff
@@ -47,13 +47,13 @@ your client and daemon API versions.
 
 > This command is experimental.
 >
-> This  command is experimental on the Docker client. It should not be used in
+> This command is experimental on the Docker client. It should not be used in
 > production environments.
 > To enable experimental features in the Docker CLI, edit the
 > [config.json](/engine/reference/commandline/cli.md#configuration-files)
-> and set `experimental` to `enabled`.
->
-> {% include experimental.md %}
+> and set `experimental` to `enabled`. You can go [here](https://github.com/docker/docker.github.io/blob/master/_includes/experimental.md)
```

Comment:
I'm a bit confused why we're linking to GitHub for this one, instead of to a page at docs.docker.com

Reply:
I'm linking it there because the page at docs.docker.com lives on the CLI master repo so it hasn't been published yet. Once the changes go live, I'll link it to there.

Reply:
Ah, gotcha; but I thought we had the other include that has the same content https://github.com/docker/docker.github.io/blob/master/_includes/experimental-feature.md

Reply:
yeah I'm not sure There's not much of a difference between them. I think the one you linked was made specifically for desktop. I'm not sure though, maybe @usha-mandya might have a better idea.

Reply:
Looks like on the CLI reference, we use an include that looks like this; https://docs.docker.com/engine/reference/commandline/deploy/

<img width="918" alt="Screenshot 2019-10-01 at 21 11 47" src="https://user-images.githubusercontent.com/1804568/65992763-2513f800-e490-11e9-8546-84bbb0656c92.png">


And there we link to the knowledgeable for a description of Experimental; https://success.docker.com/article/experimental-features

Reply:
Ah, dang, looks like some pages are labeled "_experimental_" and others are labeled "_experimentalcli_".

I can make _experimental_ and _experimentalcli_ the same or go into those pages and assign them _experimentalcli_, in case we want _experimental_ to be for another use case.

https://docs.docker.com/engine/reference/commandline/checkpoint/ is another with the same window.

Reply:
experimental is for those features that are enabled on the daemon side, whereas experimentalcli is for those that are enabled on the cli

Reply:
oh I see. Besides that, are there specific differences between them? How they're enabled or anything else?

Reply:
@adrian-plata `experimental-features.md` was the first version of the experimental features blurb. I see it only used in the Mac FAQs topic. I can update this topic to use `experimental.md` so we don't have to maintain two files. 

Agree with @thaJeztah on GitHub linking. We should point users to our docs instead. There is a new node for Experimental features in the TOC update we are working on. Wondering if we can point users to an overview page here instead of linking to GitHub when the changes are live. Thoughts?

Reply:
Adding a pointer to the cli experimental information section. https://github.com/docker/docker.github.io/pull/9569 

If there are no differences, it makes sense to update the pages so they're the same. Eventually, we'll probably want to use one if that's the case.